wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
 
 # Privacy Statement
 
-Please note that the English version of this privacy statement is the original and, as such, the binding version. To access the English version, go to the top/top right of this page and click on the "en" link.
-The following only applies to the version you can download from this website, [jamulus.io](https://jamulus.io). 
+**The English version of this document represents the official privacy statement for the _current version_ of the software available on this website. If there is a conflict between the English language version and any translation, the English version shall apply. Earlier versions may have had different privacy terms which are no longer applicable or supported. Users are encouraged to use the most current version of the Jamulus software.**
+ 
 
 ## Definition of Terms
```

Comment:
If this page intentionally _only_ applies to users of the _client_, that needs to be made clear very early, not just left for the reader to work out from what's being said.

Reply:
I think this privacy statement applies to the current version of both client and server software.

Reply:
There is only one piece of software.  How you use it is up to you but the the terms and conditions apply to that _one piece_ of software.